CHANGES

```diff
@@ -26,17 +26,17 @@ Upcoming release: BREAKING
 
 ## Changes
 
- * Added support for the ARM Cortex A55
- * Added support for the ODroid C4
- * Added support for the Avnet MaaXBoard
- * Added support for arm_hyp on qemu-arm-virt platfrom with cortex-a15 CPU
- * Added support for qemu-riscv-virt
- * Rename libsel4 config option ENABLE_SMP_SUPPORT to CONFIG_ENABLE_SMP_SUPPORT to be namespace compliant.
- * Rename libsel4 config option AARCH64_VSPACE_S2_START_L1 to CONFIG_AARCH64_VSPACE_S2_START_L1 to be namespace
-   compliant.
+* Added support for the ARM Cortex A55
+* Added support for the ODroid C4
+* Added support for the Avnet MaaXBoard
+* Added support for arm_hyp on qemu-arm-virt platfrom with cortex-a15 CPU
+* Added support for qemu-riscv-virt
+* Rename libsel4 config option ENABLE_SMP_SUPPORT to CONFIG_ENABLE_SMP_SUPPORT to be namespace compliant.
+* Rename libsel4 config option AARCH64_VSPACE_S2_START_L1 to CONFIG_AARCH64_VSPACE_S2_START_L1 to be namespace
+  compliant.
 * Remove imx31/kzm platform support. This platform is being removed as it is sufficiently old and unused.
 * Remove ARM1136JF_S and ARMv6 support. This architecture version is being removed as it is sufficiently old and
-  unused. See https://sel4.atlassian.net/browse/RFC-8.
+  unused. See <https://sel4.atlassian.net/browse/RFC-8.>
```

Comment:
Including the final full stop seem wrong. Also, why apply the `<...>` style here for links, shouldn't this be something like `[RFC-8](https://sel4.atlassian.net/browse/RFC-8).` to match the markdown style for the other links in this file?

Reply:
Yes, you're right, thanks for spotting that, the `.` should definitely be outside.

And `[RFC-8](https://sel4.atlassian.net/browse/RFC-8)` is probably nicer, I just removed the linter warnings I got. Will update.